storybook/src/components/StandaloneLink/StandaloneLink.stories.tsx

```diff
@@ -61,26 +62,23 @@ export const InverseColour: Story = {
   },
 }
 
-export const InAnArticle: Story = {
+export const WithChevronIcon: Story = {
   args: {
-    children: 'Alle bouw- en verkeerswerkzaamheden',
+    children: 'Bekijk alle werkzaamheden',
+    icon: ChevronForwardIcon,
   },
   decorators: [
     (Story) => (
-      <Grid>
-        <Grid.Cell span={{ narrow: 4, medium: 5, wide: 6 }}>
-          <article>
-            <Heading className="ams-mb-xs" level={2}>
-              Werkzaamheden
-            </Heading>
-            <Paragraph className="ams-mb-s">
-              Lees waar en wanneer we werken aan nieuwbouw, groot onderhoud, herinrichting van straten en wegen, aanpak
-              van parken of ontwikkeling van hele gebieden.
-            </Paragraph>
-            <Story />
-          </article>
-        </Grid.Cell>
-      </Grid>
+      <article style={{ maxWidth: '32rem' }}>
```

Comment:
You could've used the polymorphic grid (`as="article"`) and I kind of liked the grid example, I think it's useful to our users. The inline style is not something we encourage right?

Reply:
People can’t see and copy the inline style because it’s in a decorator. We agreed that this is fine, and that we’re glad to lose the indentation whitespace that Grid introduced.